packages/faustwp-cli/src/telemetry/marshallTelemetryData.ts

```diff
@@ -19,6 +19,8 @@ export interface TelemetryData {
   node_faustwp_core_version?: string;
   node_faustwp_cli_version?: string;
   node_faustwp_blocks_version?: string;
+  node_faustwp_block_editor_utils_version?: string;
+  node_faustwp_experimental_app_router_version?: string;
```

Comment:
Nice! Could we also include this data in the related REST API endpoint for telemetry:

https://github.com/wpengine/faustjs/blob/telemetry-updates/plugins/faustwp/includes/rest/callbacks.php#L216-L239